mon-pix/tests/integration/components/qrocm-proposal-test.js

```diff
@@ -88,5 +88,79 @@ describe('Integration | Component | QROCm proposal', function() {
         });
       });
     });
+
+    [
+      { proposals: '${input}', expectedAriaLabel: ['Réponse 1'] },
+      { proposals: '${rep1}, ${rep2} ${rep3}', expectedAriaLabel: ['Réponse 1', 'Réponse 2', 'Réponse 3'] },
+      { proposals: 'Réponses :↵${rep1}\n${rep2}', expectedAriaLabel: ['Réponse 1', 'Réponse 2'] },
+      { proposals: 'Vidéo : ${video#.ex1} ${video2#.ex2}\nImage : ${image} ${image2}', expectedAriaLabel: ['Réponse 1', 'Réponse 2', 'Réponse 3', 'Réponse 4'] },
+      { proposals: 'Le protocole ${https} assure que la communication entre l\'ordinateur d\'Adèle et le serveur de la banque est ${sécurisée}.', expectedAriaLabel: ['Réponse 1', 'Réponse 2'] },
+      { proposals: '- ${NumA} Il classe les pages trouvées pour les présenter\n- ${NumB} Il sélectionne  les pages correspondant aux mots', expectedAriaLabel: ['Réponse 1', 'Réponse 2'] },
+    ].forEach((data) => {
+      describe(`Component aria-label accessibility when proposal is ${data.proposals}`, function() {
+        beforeEach(async function() {
+          // given
+          this.set('proposals', data.proposals);
+          this.set('answerValue', '');
+          this.set('format', 'phrase');
+
+          // when
+          await render(hbs`{{qrocm-proposal proposals=proposals format=format answerValue=answerValue}}`);
+        });
+
+        it('should have an aria-label', async function() {
+          // then
+          expect(findAll('.challenge-response__proposal').length).to.be.above(0);
+          findAll('.challenge-response__proposal').forEach((element, index) => {
+            expect(element.getAttribute('aria-label')).to.equal(data.expectedAriaLabel[index]);
+          });
+        });
+
+        it('should not have a label', async function() {
+          // then
+          expect(find('label')).to.be.null;
+        });
+      });
+    });
+
+    [
+      { proposals: 'texte : ${rep1}\nautre texte : ${rep2}', expectedLabel: ['texte :', 'autre texte :'] },
+    ].forEach((data) => {
+      describe(`Component label accessibility when proposal is ${data.proposals}`, function() {
+
+        beforeEach(async function() {
+          // given
+          this.set('proposals', data.proposals);
+          this.set('answerValue', '');
+          this.set('format', 'phrase');
+
+          // when
+          await render(hbs`{{qrocm-proposal proposals=proposals format=format answerValue=answerValue}}`);
+        });
+
+        it('should have a label', async function() {
+          // then
+          expect(findAll('label').length).to.be.above(0);
```

Comment:
to.equal(taille du tableau expected)
sortir const = findAll('.challenge-response__proposal')